plugin.rb

```diff
@@ -42,7 +42,7 @@
 
   add_to_serializer(:group_show, :assignment_count) do
     Topic.joins("JOIN topic_custom_fields tcf ON topics.id = tcf.topic_id AND tcf.name = 'assigned_to_id' AND tcf.value IS NOT NULL")
-      .where("tcf.value IN (SELECT group_users.user_id::varchar(255) FROM group_users WHERE (group_id IN (SELECT id FROM groups WHERE name = ?)))", object.name).count
+      .where("tcf.value IN (SELECT group_users.user_id::varchar(255) FROM group_users WHERE (group_id IN (SELECT id FROM groups WHERE name = ?))) AND topics.deleted_at IS NULL", object.name).count
```

Comment:
Can we split this into multiple `where` calls as well? It is pretty hard to read at the moment